frontends/election-manager/src/components/smartcard_modal/status_message.tsx

```diff
@@ -126,28 +126,25 @@ export function InProgressStatusMessage({
   actionStatus,
 }: InProgressStatusMessageProps): JSX.Element | null {
   const { action } = actionStatus;
-  const actionRoleReadableString = userRoleToReadableString(actionStatus.role);
 
   let text: string;
   switch (action) {
     case 'Program': {
-      text = `Creating ${actionRoleReadableString} Card`;
+      text = 'Programming card';
```

Comment:
Consciously using "Programming" here instead of "Creating" since "Creating card" sounds a bit funny, like we're manufacturing a physical card LOL (when we had the user role here, e.g. "Creating Poll Worker Card", it sounded okay)